### PR TITLE
Update tutorials to ArbSDK v4 (5)

### PR DESCRIPTION
**Pull request by TucksonDev**
This PR updates the following tutorials:

- outbox-execute
- l1-l3-teleport
- l2-block-verification-in-assertion (renamed to `block-verification-in-parent-chain-assertion`)

This PR also includes minor fixes in some other tutorials:
- Add possibility to register multiple custom networks (needed for the l1-l3-teleport tutorial)
- Load dotenv config in all tutorials (some were missing) to be able to load the local .env file too

To test l1-l3-teleport on a nitro-testnode, you can find instructions in [this Notion page](https://www.notion.so/arbitrum/How-to-test-the-teleporter-in-nitro-testnode-14a01a3f59f880d895c9e2ce9702add4?pvs=4)

### CODE REVIEW

File: customNetwork.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "bridge": "0x5eCF728ffC5C5E802091875f96281B5aeECf6C49",
     "inbox": "0x9f8c1c641336A371031499e3c362e40d58d0f254",
     "outbox": "0x50143333b44Ea46255BEb67255C9Afd35551072F",
-    "rollup": "0x837Fadd53667eFCE195D5B75063aaB277f7d712C",
+    "rollup": "0x7d98BA231d29D5C202981542C0291718A7358c63",
```
Comment (TucksonDev) [inline customNetwork.json:8]: Changed in the last commit of the `release` branch

File: packages/outbox-execute/scripts/exec.js [hunk 2]
```diff
@@ -1,78 +1,122 @@
...
-  console.log('Done! Your transaction is executed', rec)
+  const executeTransaction = await childToParentMessage.execute(
+    childChainProvider
+  )
+  const executeTransactionReceipt = await executeTransaction.wait()
+  console.log('Done! Your transaction is executed', executeTransactionReceipt)
+}
+
+// Getting the transaction hash from the command arguments
+if (process.argv.length < 3) {
```
Comment (Jason-W123) [inline packages/outbox-execute/scripts/exec.js:106]: Same here

File: packages/block-verification-in-parent-chain-assertion/scripts/exec.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  if (lastChildChainBlockNumber > childChainBlockNumberToVerify) {
+    console.log(`${childChainBlockNumberToVerify} has been processed as part of the latest ${useCreatedNodeInsteadOfConfirmed ? 'created' : 'confirmed'} RBlock/node`)
+  } else {
+    console.log(`${childChainBlockNumberToVerify} has NOT been processed as part of the latest ${useCreatedNodeInsteadOfConfirmed ? 'created' : 'confirmed'} RBlock/node`)
+  }
+  console.log(`************`)
+};
+
+// Getting the transaction hash from the command arguments
+if (process.argv.length < 3) {
```
Comment (Jason-W123) [inline packages/block-verification-in-parent-chain-assertion/scripts/exec.ts:91]: How about we also use `yargs` to process the arg here?

**Review by Jason-W123 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**